.github/workflows/ci-core.yml

```diff
@@ -97,6 +97,8 @@ jobs:
     if: github.actor != 'dependabot[bot]'
     needs: [filter]
     runs-on: ubuntu-latest-64cores-256GB
+    outputs:
+      job_id: ${{ github.job }}
```

Comment:
This isn't printing out the job ID, it's printing out `core` :facepalm: https://github.com/smartcontractkit/chainlink/actions/runs/8905733446/job/24457135451?pr=13063#step:14:25

Reply:
We can get the run attempt from env here: https://github.com/smartcontractkit/push-gha-metrics-action-source/blob/89490946f3d4bad6d0b5103686e42bdb25bf323d/src/context.ts#L77

From there, we can get the Job ID via a http request, or just link the attempt and call it good enough from there.